compiler/src/dotty/tools/dotc/parsing/Scanners.scala

```diff
@@ -1157,6 +1158,8 @@ object Scanners {
               nextChar()
             }
           }
+          val alt = if (oct == LF) "\\n" else "\\u%04x" format oct
```

Comment:
`if oct == LF then raw"\n" else f"\u$oct%04x"` for style points?

Reply:
👍 fixed